map/src/context/AppContext.js

```diff
@@ -356,7 +358,10 @@ export const AppContextProvider = (props) => {
         routeMode, setRouteMode,
         weatherPoint, setWeatherPoint,
         routeTrackFile, setRouteTrackFile,
-        searchCtx, setSearchCtx
+        searchCtx, setSearchCtx,
+        favoriteFile, setFavoriteFile,
```

Comment:
looks like too many state variables for such top level and too much of internal structure exposed